src/forms/staff/forms.jsx

```diff
@@ -0,0 +1,126 @@
+import { LoaderCircle } from 'lucide-react';
+import { Button } from '@/components/ui/button';
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form';
+import { Input } from '@/components/ui/input';
+import { DropdownMenu, DropdownMenuContent, DropdownMenuItem, DropdownMenuTrigger } from '@/components/ui/dropdown-menu';
+import { PasswordInput } from '@/components/forms/password-input';
+import { specialtiesWithLabels } from '@/utils/utils';
+
+export function RegisterStaffForm({ form, onSubmit, isLoading, error }) {
+
+
+  return (
+    <Form {...form}>
+      <form onSubmit={form.handleSubmit(onSubmit)} className="space-y-8">
```

Comment:
To use two columns on desktop or big screens

```suggestion
      <form onSubmit={form.handleSubmit(onSubmit)} className="grid lg:grid-cols-2 gap-x-8 gap-y-8">
```